src/test/java/hudson/plugins/ec2/ConfigurationAsCodeTest.java

```diff
@@ -34,13 +34,14 @@ public class ConfigurationAsCodeTest {
     @Rule
     public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
 
-    @Test
-    @ConfiguredWithCode("EC2CloudEmpty.yml")
-    public void testEmptyConfig() throws Exception {
-        final AmazonEC2Cloud ec2Cloud = (AmazonEC2Cloud) Jenkins.get().getCloud("");
-        assertNotNull(ec2Cloud);
-        assertEquals(0, ec2Cloud.getTemplates().size());
-    }
+    // Jenkins 2.403: Cloud Nodes cannot be created with an empty name
```

Comment:
Unable to catch the illegal argument exception when the cloud node is created. Wasn't sure if there was a way to do that for this particular test

Reply:
Maybe you can check the instance logs and search for the exception there?. More important, is this change in behaviour expected or a side effect?

Reply:
But isn't the point of the test to just create an empty cloud name? The behavior is now blocked in the cloud implementation